drivers/adc/CMakeLists.txt

```diff
@@ -29,5 +29,4 @@ zephyr_library_sources_ifdef(CONFIG_ADC_ADS1X1X		adc_ads1x1x.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_GD32		adc_gd32.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_ADS1119		adc_ads1119.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_RPI_PICO	adc_rpi_pico.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_XMC4XXX		adc_xmc4xxx.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_ESP32		adc_esp32.c)
```

Comment:
Believe these lines should not be removed

Reply:
Replaced these lines